code/modules/power/singularity/boh_tear.dm

```diff
@@ -22,17 +16,29 @@
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
 	flags_1 = SUPERMATTER_IGNORES_1
 
-/obj/boh_tear/Initialize(mapload)
-	. = ..()
-	QDEL_IN(src, 5 SECONDS) // vanishes after 5 seconds
-
+/obj/boh_tear/proc/start_disaster()
+	apply_wibbly_filters(src)
+	playsound(loc, 'sound/effects/clockcult_gateway_disrupted.ogg', vary = 200, extrarange = 3, falloff_exponent = 1, frequency = 0.33, pressure_affected = FALSE, ignore_walls = TRUE, falloff_distance = 7)
 	AddComponent(
 		/datum/component/singularity, \
-		consume_range = BOH_TEAR_CONSUME_RANGE, \
-		grav_pull = BOH_TEAR_GRAV_PULL, \
+		consume_range = 1, \
+		grav_pull = 21, \
 		roaming = FALSE, \
 		singularity_size = STAGE_SIX, \
 	)
+	addtimer(CALLBACK(src, PROC_REF(bagulo_time)), 9 SECONDS, TIMER_DELETE_ME)
```

Comment:
shouldn't this be 10 seconds?

Reply:
or do you want it to delete like as it fades kinda thing (so like 9.5)

Reply:
See above